src/main/java/db/Storage.java

```diff
@@ -0,0 +1,28 @@
+package db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final String NOT_NULL_MESSAGE = "Null can not be add";
+    private static final Map<String, Integer> storage = new HashMap<>();
+
+    public static Integer getFruitsNumber(String fruit) {
+        if (storage.isEmpty() || !storage.containsKey(fruit)) {
+            return 0;
+        }
+        return storage.get(fruit);
+    }
+
+    public static void storeFruit(String fruit, Integer amount) {
+        if (fruit == null) {
+            throw new RuntimeException(NOT_NULL_MESSAGE);
+        }
+        storage.put(fruit, amount);
+    }
+
+    public static Map<String, Integer> getStorage() {
+        return storage;
+    }
```

Comment:
I think it would be better if the work with the storage was in a separate file like "StorageDao"